ui/components/layouts/dashboard.js

```diff
@@ -31,12 +33,16 @@ function Dashboard ({ children }) {
   ]
 
   const subNavigation = [
-    { name: 'Settings', href: '/settings', admin: true }
+    { name: 'Settings', href: '/settings', admin: accessToSettingsPage }
   ]
 
   // redirect non-admin routes if user isn't admin
   for (const n of [...navigation, ...subNavigation]) {
```

Comment:
I think we can remove this for loop (and the if statement within it) for now, we only have one subNavigation and it's `/settings`